### PR TITLE
448 creating arrivals

### PR DESCRIPTION
**Pull request by pezholio**
This adds a screen to check a person has arrived. If they have, we record the date they arrived, and the date we expect them to depart, together with some notes.

# Screenshots

image

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Creating arrivals](https://trello.com/c/ZxnaGH3p/448-creating-arrivals)

File: integration_tests/e2e/arrivals.cy.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  it('creates an arrival', () => {
+    // Given I am logged in
+    cy.signIn()
+
+    // And I have a booking for a premises
+    const premises = premisesFactory.build()
+    const bookingId = 'some-uuid'
+    const arrival = arrivalFactory.build()
+
+    cy.task('subSinglePremises', premises)
```
Comment (richpjames) [inline integration_tests/e2e/arrivals.cy.ts:23]: haven't had chance to look through this properly yet but this should be `stubSinglePremises`

Comment (richpjames) [reply integration_tests/e2e/arrivals.cy.ts:23]: I also can't find where the task is defined

File: wiremock/stubApis.ts [hunk 2]
```diff
@@ -39,10 +40,29 @@ premises.forEach(p => {
...
 
+stubs.push(async () =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPathPattern:
+        '/premises/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/bookings/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/arrivals',
+    },
+    response: {
+      status: 200,
```
Comment (richpjames) [inline wiremock/stubApis.ts:51]: should this be a `201`?

**Review by richpjames — APPROVED** [review]
LGTM 🚢

File: wiremock/stubApis.ts (hunk 2 above)
Comment (pezholio) [reply wiremock/stubApis.ts:68]: Whoops - yeah! I think that was down to a bad merge.

File: integration_tests/e2e/arrivals.cy.ts (hunk 1 above)
Comment (pezholio) [reply integration_tests/e2e/arrivals.cy.ts]: Was a typo here https://github.com/ministryofjustice/approved-premises-ui/blob/02b0f0d31d754f78b58f81cd7d903f4284f3d25e/integration_tests/mockApis/premises.ts#L21 All fixed now!

**Pull request merged**